akka-actor-tests/src/test/scala/akka/routing/ScatterGatherFirstCompletedSpec.scala

```diff
@@ -86,4 +89,15 @@ class ScatterGatherFirstCompletedSpec extends AkkaSpec with DefaultTimeout with
 
   }
 
+  "Scatter-gather pool" must {
+
+    "without routees should reply immediately" in {
+      val probe = TestProbe()
+      val router = system.actorOf(ScatterGatherFirstCompletedPool(nrOfInstances = 0, within = 5.seconds).props(Props.empty))
```

Comment:
There might be corner cases with not allowing 0, e.g. could work with 0 + resizer.